### PR TITLE
Newsweek

### PR DESCRIPTION
**Pull request by Ramon-Carrillo**
Worked solo on this project. I finished the project and applied Bootstrap, HTML, and CSS. I would like you to make a review of my work.

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: INVALID :yellow_circle: 

Please follow the instructions to add the specific linters for the repository. Here is the link:

- [ ] [html-css](https://github.com/microverseinc/linters-config)

Please work on these things also:

- [ ] Professional repository, the one the project has is not descriptive enough.
- [ ] Better Pull Request message

## Happy coding! :smiley:

**Comment by Ramon-Carrillo** [comment]
Added linters ask requested for a valid pull request review.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning::vertical_traffic_light: 

### Hello @Ramon-Carrillo :wave: 

**I must say, you've done an awesome job right here :clap:**
- Your linters are green :green_circle: 
- Good understanding of the project :heavy_check_mark: 

**In summary, this project is looking good and is almost complete. Talking about the completeness, there are some issues we need to fix before this project can be approved. Let's get to it:**

**[ REQUIRED CHANGES ]**
_As the main point of this project is to use Bootstrap, I will focus on this aspect in this code review_
- [ ] Your readme is not detailed enough. Check my comments below on what to do to make it better.



- [ ] You used a lot of CSS bad practices in your project.
- You have a lot of repetitions in your CSS file. Kindly review your CSS rules and group the common blocks together.
- You overused the `!important` rule. Kindly find another way to make things work without the rule.
- Some of the duplications are highlighted below. Fix them and the rest.



- [ ] This is a bootstrap project. You are expected to use bootstrap classes as much as possible. 
- You can drastically reduce the size of your CSS file by using bootstrap classes for rules like margins, paddings, colors, and other styles.
- Using media queries in this project is not bad, but if bootstrap can do it, you should use bootstrap.



- [ ] Your navigation bar should contain the current date
__Original navbar__


__Your navbar__


- [ ] Adjust newsletter section for medium screen devices 

Original page



Your clone



**Small devices**
- [ ] The top stories section is followed by the featured stories section in the original. This is not the case in your clone page. Kindly rearrange it.

- [ ] Kindly re-structure your more stories section

Original page


Your clone



- [ ] Editors' pick section also needs some re-adjustments

Original page


Your clone 



**Please note: You are asked to be more precise and to make sure that your page looks exactly the same as the source page.**

Kindly make the required changes and request for another review when you are done.

*__You can reach out to me on slack (@taofeekolalere) or GitHub (@teekaytech).__*

**Happy coding. :man_technologist:**

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,804 @@
...
+  background-color: white;
+  font-size: 16px;
+  outline: none;
+  border: none;
+  border-radius: 0;
+  z-index: 10;
+}
+
+.search-icon {
+  visibility: visible !important;
```
Comment (umairarshadbutt) [inline css/style.css:278]: Remove `!important`

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+    height: 265px;
+  }
+}
+
+/* Medium devices (tablets, 768px and up) ---------------------------- */
+@media (min-width: 768px) {
+  .sticky2 {
+    position: sticky !important;
+    bottom: 5rem !important;
+    align-self: flex-end !important;
```
Comment (umairarshadbutt) [inline css/style.css:629]: Remove `!important`

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+  }
+}
+
+/* Large devices (desktops, 992px and up)---------------------------- */
+@media (min-width: 992px) {
+  .sticky1,
+  .sticky2 {
+    position: sticky !important;
+    bottom: 15rem !important;
+    align-self: flex-end !important;
```
Comment (umairarshadbutt) [inline css/style.css:765]: Remove `!important`

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Ramon-Carrillo
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

- [x] Follow the adustment suggestions
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)

- [x] make sure the order of sections on small screens



### Navbar (medium screen)
- [x] Replace **SIGN IN** with user icon
- [x] Remove the search text from navbar

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://ramon-carrillo.github.io/Newsweek/)|
|:---:|:---:|
|source-page|clone|



### Navbar (small screen)
- [x] Replace **SIGN IN** with user icon
- [x] Display **SUBSCRIBE** button

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://ramon-carrillo.github.io/Newsweek/)|
|:---:|:---:|
|source-page|clone|

### SUBSCRIBE
- [ ] Text should be overflow on images in small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://ramon-carrillo.github.io/Newsweek/)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by Ramon-Carrillo** [comment]
Made adjustments that were requested in the last pull request. I added a user icon, fixed breakpoint, and reorder the order.

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

- [ ] Optional Please add some description in about section of your repo

- [ ] Please add all images and CSS files in a single folder

- [ ] Please add more description in the readme file. 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**